pkg/ccl/benchccl/rttanalysisccl/testdata/benchmark_expectations

```diff
@@ -11,9 +11,9 @@ exp,benchmark
 14,AlterSurvivalGoals/alter_empty_database_from_zone_to_region
 24,AlterSurvivalGoals/alter_populated_database_from_region_to_zone
 24,AlterSurvivalGoals/alter_populated_database_from_zone_to_region
-25,AlterTableLocality/alter_from_global_to_rbr
-16,AlterTableLocality/alter_from_global_to_regional_by_table
-13,AlterTableLocality/alter_from_rbr_to_global
-13,AlterTableLocality/alter_from_rbr_to_regional_by_table
-16,AlterTableLocality/alter_from_regional_by_table_to_global
-25,AlterTableLocality/alter_from_regional_by_table_to_rbr
+28,AlterTableLocality/alter_from_global_to_rbr
+19,AlterTableLocality/alter_from_global_to_regional_by_table
+18,AlterTableLocality/alter_from_rbr_to_global
+18,AlterTableLocality/alter_from_rbr_to_regional_by_table
+19,AlterTableLocality/alter_from_regional_by_table_to_global
+28,AlterTableLocality/alter_from_regional_by_table_to_rbr
```

Comment:
It's not obvious to me why the number of RTs went up for these tests, especially when they went down in the non-ccl tests. @fqazi do you have any ideas?

Reply:
@mgartner This is happening because we are getting the leased database descriptor inadvertently after the change. i.e. The getter in schemaResolver.LookupObject is setup like this:

```
	b := sr.descCollection.ByName(sr.txn)
	if !sr.skipDescriptorCache && !flags.RequireMutable {
		// The caller requires this descriptor to *not* be leased,
		// so lets assert this here. Normally the planner / resolver
		// will propagate flags so we don' need to check on a
		// look up level.
		if flags.AssertNotLeased {
			return false, prefix, nil,
				errors.AssertionFailedf("unable to get leased descriptor for (%q), resolver was not configured properly", obName)
		}
		b = sr.descCollection.ByNameWithLeased(sr.txn)
	}
```

So, normally when resolving the table name this code would just get the database descriptor from KV.

Reply:
This is minor and we can ignore it. I think this is just the difference from the flags being used in lookupDatabase and impacts DDL, which is already fairly slow (i.e. needing backfills, etc..).
<!-- Reviewable comment -OE0xr5T3LB5I6tlKOTY:bbuo3wk -->
<!-- Sent from Reviewable.io -->